### PR TITLE
My better version of hello world message using namespace

### PR DESCRIPTION
**Pull request by lvallone3315**
Please review my changes

### CODE REVIEW

File: GitExample/GitExample.cpp [hunk 1]
```diff
@@ -3,10 +3,13 @@
 
 #include <iostream>
 
+using namespace std;
+
```
Comment (leefdu) [inline GitExample/GitExample.cpp:7]: Please use a narrower "using" - for example "using std;"

File: GitExample/GitExample.cpp [hunk 2]
```diff
@@ -3,10 +3,13 @@
...
 int main()
 {
     std::cout << "Hello World!\n";
     std::cout << "  Hello from GitExample_branch1!\n\n\n";
+    cout << "Yet another hello from me\n";
 }
 
 // Run program: Ctrl + F5 or Debug > Start Without Debugging menu
```
Comment (leefdu) [inline GitExample/GitExample.cpp]: I know you didn't change this - but really should remove the boilerplate.

**Review by leefdu — CHANGES_REQUESTED** [review]
Please change as suggested in the review

**Review by leefdu — APPROVED** [review]
Looks good - please merge into master

**Comment by lvallone3315** [comment]
Merge my better cout

**Pull request merged**